app/kubernetes-ingress-controller/1.0.x/guides/prometheus-grafana.md

```diff
@@ -44,7 +44,9 @@ We’ll install both Prometheus and Grafana in a dedicated `monitoring` namespac
 To install Prometheus, execute the following:
 
 ```bash
-$ helm install --name prometheus stable/prometheus --namespace monitoring --values https://bit.ly/2RgzDtg --version 11.0.3
+$ kubectl create namespace monitoring
+$ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+$ helm install prometheus prometheus-community/prometheus --namespace monitoring --values https://bit.ly/2RgzDtg --version 11.0.3
```

Comment:
Why `11.0.3` which appears to be over a year old?

(applies to all copies of this)

Reply:
For the backported (older version) docs, I'm just doing the minimum to make them not completely broken (using Helm 2 syntax that doesn't exist in Helm 3 and using repo URLs that will soon go away) without trying to update for changed content. They're using the same versions they did originally to try and not break any of the other steps, but updated so that the commands still work.

The [2.0 version](https://github.com/Kong/docs.konghq.com/blob/8df786da11a3ee935ef8e9c32d0e85d6e004229b/app/kubernetes-ingress-controller/2.0.x/examples/001_multiple-services.yaml), which is new, _does_ go through and bump up to the latest version of things (which ends up being an entirely different chart/workflow), along with changes necessary in other steps to use those latest versions.

I take different approaches depending on the freshness of what I'm updating. Older stuff gets minimal revisions to try and preserve something that's as close to the original as it can be without being broken, newer stuff/latest version gets a more holistic revision across the board.